src/Operation/FindOneAndUpdate.php

```diff
@@ -59,9 +59,9 @@ public function __construct($databaseName, $collectionName, $filter, $update, ar
             throw new InvalidArgumentTypeException('$update', $update, 'array or object');
         }
 
-        if ( ! \MongoDB\is_first_key_operator($update)) {
-            throw new InvalidArgumentException('First key in $update argument is not an update operator');
-        }
+      //  if ( ! \MongoDB\is_first_key_operator($update)) {
+       //     throw new InvalidArgumentException('First key in $update argument is not an update operator');
```

Comment:
Yup. @bjori This isn't accidental though, my code wouldn't run without this, which is weird.